content/en/dashboards/widgets/geomap.md

```diff
@@ -35,8 +35,8 @@ The geomap widget visualizes geographic data with shaded regions or points. It c
   {{% tab "Regions" %}}
   |  Data source    | Notes    | 
   | --------------  | -------- |
-  |Log Events   | The group by tag must include a country ISO Code following the alpha-2 ISO format. You can use the [GeoIP Processor][1] to do this, or manually include the [tags on ingest][2]. See the [Log search documentation][3] to configure a log event query.|
-  |Metric   | The group by tag must include a country ISO Code following the alpha-2 ISO format. You can [generate metrics from ingested logs][4], or manually include the [tags on ingest][2]. See the [querying documentation][5] to configure a metric query.|
+  |Log Events   | The group by tag must include a country ISO Code or country subdivision ISO Code following the alpha-2 ISO format. You can use the [GeoIP Processor][1] to do this, or manually include the [tags on ingest][2]. See the [Log search documentation][3] to configure a log event query.|
```

Comment:
Thanks for working on updating the docs! If we mention the format, I feel like we should differentiate the formats intended for each. This suggestion is just an idea; I admit the sentence is now getting really long in this suggestion.

ref: https://en.wikipedia.org/wiki/ISO_3166-2

```suggestion
  |Log Events   | The group by tag must include a country ISO Code, following the alpha-2 ISO format, or a country subdivision ISO Code, following the ISO-3166-2 format. You can use the [GeoIP Processor][1] to do this, or manually include the [tags on ingest][2]. See the [Log search documentation][3] to configure a log event query.|
```

Reply:
Same thoughts about the next line 👇

Reply:
Ah, thanks for that! I tried to make the sentence a little shorter by using some parentheses, updated.

Reply:
Love it!